narratives/ncov_sit-rep_2020-03-27.md

```diff
@@ -0,0 +1,2734 @@
+<!-- There is NO right-side text -->
+
+
+
+
+<!-- ############ EUROPE ############# -->
+<!-- This is left-side text 4-->
+# [Updates for Europe](https://nextstrain.org/ncov/2020-03-27?d=tree,map&f_region=Europe&p=grid&r=country&c=country)
+
+* [Executive summary](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=1).
+* [Background resources](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=2).     
+* [About this data](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=3).  
+* [Updates for Europe](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=4).  
+* [Updates for North America](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=7).  
+* [Updates for Central & South America](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=11).  
+* [Updates for Asia](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=13).  
+* [Updates for Africa](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=15).  
+* [Updates for Oceania](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=17).  
+* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=19).  
+* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=20).  
+
+<!-- There is no right side text -->
+
+
+<!-- ##### slide break ##### -->
+<!-- This is left-side text 5-->
+# [Continued spread across borders in the last 3-5 weeks](https://nextstrain.org/ncov/2020-03-27?c=country&d=tree,map&dmin=2020-03-06&f_region=Europe&p=full&r=country&legend=closed)
+
+Here, we show samples collected across Europe in the last 3 weeks, colored by country.
+Notably, while there are some small clusters that are country-specific, there is also a great deal of intermingling by country.
+<br><br>
+This suggests that there has been continued spread across borders within the last 3-5 weeks.
+<br><br>
+As the recent mitigation efforts have time to take effect, we will surveil to see if cases start to cluster more discretely by country.
+<!-- There is no right side text -->
+
+
+<!-- ##### slide break ##### -->
+<!-- This is left-side text 6-->
+# [European and U.S. outbreaks are intertwined](https://nextstrain.org/ncov/2020-03-27?c=region&d=tree,map&f_region=Europe,North%20America&label=clade:A2a&m=div&p=full&legend=open)
+Across the tree, we also see lots of comingling of European and American samples, indicating that the origins of the U.S. and European outbreaks are deeply intertwined.
+<br><br>
+Artificially, this may appear to flow primarily from Europe to the U.S., rather than bidirectionally; however, we expect that this is mostly an effect of sampling bias as we currently have more data from Europe.
```

Comment:
```suggestion
Artificially, this may appear to flow primarily from Europe to the U.S., rather than bidirectionally. However, we expect this is mostly an effect of sampling bias as we currently have more data from Europe.
```